src/assessor_tools.yml

```
@@ -302,53 +417,71 @@
         install_dir: /tools/TikiTorch
         unarchive_extra_opts:
           - --strip-components=1
-    - role: bloodhound
-      vars:
-        # The password for Neo4j
-        password: "{{ lookup('aws_ssm', '/neo4j/password') }}"
     - role: burp_suite_pro
       vars:
         # The name of the AWS S3 bucket where third-party software is
         # located
         third_party_bucket_name: "{{ build_bucket }}"
         # License Burp Suite Pro for the VNC user
         users:
-          - "{{ lookup('aws_ssm', '/vnc/username') }}"
+          - "{{ vnc_username }}"
+    - role: metasploit
     - role: server_setup
+    - role: visual_studio_code
 
   tasks:
     - name: Install system packages for tools where possible
       ansible.builtin.package:
         name:
-          - crackmapexec
+          - cifs-utils
           - dnsutils
           - exploitdb
           - eyewitness
+          - firefox-developer-edition-en-us-kbx
           - flameshot
           - gobuster
-          - kerberoast
           - libreoffice
-          - mimikatz
+          - nfs-common
           - nikto
-          - powershell-empire
           - powersploit
-          - responder
-          - seclists
-          - sqlmap
           - sublist3r
+          - terminator
+          - testssl.sh
           - veil
 
+    # The kaboxer group is created via the
+    # firefox-developer-edition-en-us-kbx package, and the vnc user
+    # must be a member of this group to run FIrefox Developer Edition.
+    - name: Add vnc user to kaboxer group
```

Fix typo and use consistent capitalization with the rest of this file:
```suggestion
    # firefox-developer-edition-en-us-kbx package, and the VNC user
    # must be a member of this group to run Firefox Developer Edition.
    - name: Add VNC user to kaboxer group
```